docs/release-guide.md

```diff
@@ -158,11 +158,11 @@ Committed revision 37434.
 
 ### Make source code package and upload to Apache SVN
 
-Here's an example of preparing a 1.0-rc1 version. Before you make package, make sure you have branch v1.0 ready on github.
+Here's an example of preparing a 1.0 version. Before you make package, make sure you have branch v1.0 ready on github.
 
 ```sh
-# Create a new version number directory and enter, for example: 1.0-rc1
-$ mkdir 1.0-rc1 && cd 1.0-rc1
+# Create a new version number directory and enter, for example: 1.0
+$ mkdir 1.0 && cd 1.0
```

Comment:
please add version to ENV, so others can use these cons lines directly

Reply:
Done